argparse/argparse_cwl_translation.py

```diff
@@ -125,4 +125,4 @@ def get_cwl_type(py_type):
         elif type(py_type).__name__ == 'builtin_function_or_method' or type(py_type).__name__ == 'FileType':
             return 'File'
         else:  # type given as a string: 'str', 'int' etc.
-            return PY_TO_CWL_TYPES[py_type]
+            return PY_TO_CWL_TYPES.get(py_type,None)
```

Comment:
Fixed in https://github.com/erasche/argparse2tool/commit/e1fb6428ba5a1f91a8ac1d1ecc88ceb5417beab7